x/concentrated-liquidity/tick.go

```diff
@@ -17,6 +17,13 @@ func (k Keeper) initOrUpdateTick(ctx sdk.Context, poolId uint64, tickIndex int64
 		return types.PoolNotFoundError{PoolId: poolId}
 	}
 
+	pool, err := k.getPoolById(ctx, poolId)
```

Comment:
I think we can remove the call to `k.PoolExists` above. `k.getPoolId` should error as well if it fails to get the pool

Reply:
great catch! fixed